lib/iris/tests/unit/experimental/ugrid/test_load_mesh.py

```diff
@@ -0,0 +1,54 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+"""
+Unit tests for the :func:`iris.experimental.ugrid.load_mesh` function.
+
+"""
+
+# Import iris.tests first so that some things can be initialised before
+# importing anything else.
+import iris.tests as tests  # isort:skip
+
+from unittest import mock
+
+from iris.experimental.ugrid import PARSE_UGRID_ON_LOAD, load_mesh
+
+
+class Tests(tests.IrisTest):
+    # All 'real' tests have been done for load_meshes(). Here we just check
+    #  that load_mesh() works with load_meshes() correctly, using mocking.
+    def setUp(self):
+        patcher = mock.patch("iris.experimental.ugrid.load_meshes")
+        self.addCleanup(patcher.stop)
```

Comment:
`patcher = mock.patch ... ; self.addCleanup(patcher.stop) ... ; patcher.start() `
All this can be replaced with use of `self.patch()`, which does all those things  (provided in `IrisTest`).

Reply:
436f5f737bc1c1b8fc08631717bcf64fab5f054e